src/app/core/file/edit-file/edit-file.component.ts

```diff
@@ -0,0 +1,86 @@
+import { Component } from "@angular/core";
+import { EditComponent } from "../../entity-components/entity-utils/dynamic-form-components/edit-component";
+import { DynamicComponent } from "../../view/dynamic-components/dynamic-component.decorator";
+import { FileService } from "../file.service";
+import { EntityMapperService } from "../../entity/entity-mapper.service";
+import { HttpEventType } from "@angular/common/http";
+import { ProgressSpinnerMode } from "@angular/material/progress-spinner";
+import { AlertService } from "../../alerts/alert.service";
+import { LoggingService } from "../../logging/logging.service";
+import { ConfirmationDialogService } from "../../confirmation-dialog/confirmation-dialog.service";
+
+@DynamicComponent("edit-file")
```

Comment:
```suggestion
@DynamicComponent("EditFile")
```
other components have used CamelCase